btc/path_transaction.go

```diff
@@ -0,0 +1,107 @@
+package btc
+
+import (
+	"context"
+	"encoding/hex"
+	"errors"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathTransaction(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "transaction/" + framework.GenericNameRegex("name"),
+		Fields: map[string]*framework.FieldSchema{
+			"name": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "Wallet name",
+			},
+			"rawTx": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "Raw transaction to be signed",
+			},
+			"multisig": &framework.FieldSchema{
+				Type:        framework.TypeBool,
+				Description: "Multisig transaction",
+				Default:     false,
+			},
+		},
+		Callbacks: map[logical.Operation]framework.OperationFunc{
+			logical.UpdateOperation: b.pathTransactionWrite,
+		},
+
+		HelpSynopsis:    pathTransactionHelpSyn,
+		HelpDescription: pathTransactionHelpDesc,
+	}
+}
+
+func (b *backend) pathTransactionWrite(ctx context.Context, req *logical.Request, d *framework.FieldData) (*logical.Response, error) {
+	walletName := d.Get("name").(string)
```

Comment:
enforce the token check and revoke once before return tx hash

Reply:
ok, at the moment since the raw tx is not decoded, this operation returns only the signature

Reply:
should be a single use seal in any case. We should create a new token for each new operation to guarantee accountability